R/classes.R

```diff
@@ -55,9 +53,11 @@ print.PartInv <- function(x, ...) {
   cat("Cutpoint on the latent scale (xi): ", round(x$cutpt_xi, 3), "\n")
   cat("Cutpoint on the observed scale (Z): ", round(x$cutpt_z, 3), "\n")
   cat("AI ratio: ", round(x$ai_ratio, 3), "\n\n")
-  print.PartInvSummary(x$summary)
-  if (!is.null(x$plot)) {
-    print(x$plot)
+  cat("Partial invariance results:\n\n")
```

Comment:
I think it might be clearer to move the following line 
`  cat("Partial invariance results:\n\n")`
to before
 `cat("Proportion selected: ", round(x$propsel, 3), "\n")` to make the output easier to read when/if strict invariance results are shown after the partial invariance results.